src/ansys/geometry/core/plotting/plotter.py

```diff
@@ -281,6 +287,11 @@ def add_body(
         # Use the default PyAnsys Geometry add_mesh arguments
         self.__set_add_mesh_defaults(plotting_options)
         dataset = body.tessellate(merge=merge)
+        if "clip" in plotting_options:
+            plotting_options.pop("clip")
+            dataset = self.clip(dataset, **plotting_options)
+            plotting_options.pop("normal", None)
+            plotting_options.pop("origin", None)
```

Comment:
Why do you need to pop these options?

Reply:
If these are not popped, I'll get errors when passing the plotting_options to `add_mesh` function.